### PR TITLE
Add locale selector on forms index

### PR DESCRIPTION
**Pull request by zerolab**
This PR adds the locale switcher for the contrib forms index

Screenshot 2022-01-14 at 16 51 28

### CODE REVIEW

File: wagtail/contrib/forms/views.py [hunk 1]
```diff
@@ -64,13 +66,50 @@ class FormPagesListView(SafePaginateListView):
...
+        if enable_locale_filter:
+            if request.GET.get('locale'):
+                self.locale = get_object_or_404(Locale, language_code=request.GET['locale'])
+            else:
+                self.locale = Locale.get_default()
+
+        return super().get(request, *args, **kwargs)
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        local_context = {
```
Comment (kaedroho) [inline wagtail/contrib/forms/views.py:90]: nitpick: missing an `e`

**Review by kaedroho — APPROVED** [review]
This looks good to me! Just a minor typo

File: wagtail/contrib/forms/views.py (hunk 1 above)
Comment (zerolab) [reply wagtail/contrib/forms/views.py]: updated and rebased on `main` while at it

Comment (kaedroho) [reply wagtail/contrib/forms/views.py]: Ohh, I see, but thanks for changing it anyway!

**Pull request merged**